tests/DependencyInjection/ClassNotFoundException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+declare(strict_types=1);
+
+namespace Lcobucci\Chimera\Tests\DependencyInjection;
+
+final class UnableToLoadPackage extends \RuntimeException
```

Comment:
Maybe `MissingDependency` is a better name? It would be nice to have it implementing a base label interface like `Lcobucci\Chimera\Exception` so that we can organise things better